### DISCUSSION TITLE
Could we add more fields to the Webinars.AddRegistrantAsync() function?

### DISCUSSION THREAD

**Discussion by Ralpharama** [Jericho/ZoomNet#156] [Ideas]
Hi there, me again!

The webinar add participant call `POST /webinars/{webinarId}/registrants` 
https://marketplace.zoom.us/docs/api-reference/zoom-api/webinars/webinarregistrantcreate

Allows us to add more fields than just name and email , e.g. `org` and `job_title`. These would be really useful for us, as well as the others.

Maybe we could pass a user object to the `Webinars.AddRegistrantAsync()` function that could contain all the possible fields, instead of having three user params defined?

Many thanks!

**Comment by Jericho** [comment]
I agree, we definitely should add these fields. Also, I'll review the fields currently when adding a registrant to a meeting to ensure that all possible fields are available.

**Reply by Jericho** [reply]
This improvement is being tracked in #157

**Comment by Ralpharama** [comment]
Great, thanks! I'll keep an eye on the Tracker.

**Comment by Jericho** [comment]
I started working on this over the weekend and unfortunately, I can't make any progress because I am not authorized to add registrants to a meeting and/or webinar therefore it's impossible for me to test the functionality I'm adding to the library.

I submitted an application to become a Zoom ISV partner. I'll let you know when I hear back. Hopefully they'll accept me in their program and grant me access to the additional functionality.

**Reply by Ralpharama** [reply]
Appreciate your efforts!

**Comment by Jericho** [comment]
@Ralpharama Good news: @Carbon8Jim graciously granted me access to their Zoom account which has access to meeting and webinar registration. To will allow me resume working on this feature. I will be posting updates on #157.
